pkg/agent/flannel/setup.go

```diff
@@ -80,6 +86,14 @@ const (
 	ipv6
 )
 
+var (
+	netmakerBackend = `{
+		"Type": "extension",
+		"PostStartupCommand": "curl -H 'Authorization: Bearer %NETMAKERKEY%' -H 'Content-Type: application/json' -d '{\"interface\":\"cni0\",\"natenabled\":\"true\",\"ranges\":[\"'$SUBNET'\"]}'  https://api.mbuilnetmaker.ydns.eu/api/nodes/mynet/%NETMAKERNODEID%/creategateway",
+		"ShutdownCommand": "netclient disconnect && netclient leave -n mynet"
```

Comment:
user is expected to provide curl and netclient binaries on their own? Note that our k3s-root provides only a busybox curl stub that doesn't support TLS or most of the flags you're using here.

Reply:
What is `api.mbuilnetmaker.ydns.eu`?

Reply:
That was part of the old code including netmaker stuff. I removed it and just left tailscale :). Have a look at the update from 2h ago, there shouldn't be anything saying netmaker (I hope)